lib/src/builders/sentinels.dart

```diff
@@ -8,6 +8,8 @@ import 'package:nyxx/src/utils/flags.dart';
 // ASCII encoded "nyxx"
 const sentinelInteger = 0x6E797878;
 
+const sentinelDouble = double.nan;
```

Comment:
It would probably be better to choose a value other than `nan` here (`sentinelInt.toDouble()` would be fine), since nan is more likely to appear as a result of some expression that another relatively large double value.

This is especially true since the current only use of this should be limited to 0-1 (since it's the volume).

Reply:
`sentinelInt.toDouble()` isn't constant though. That's why I used NaN. Maybe `double.infinity` would be better.
Or any double above 1

Reply:
We could use `sentinelInt / 1.0` as a hack to get a constant double.

Infinity has the same problems as NaN, it might arise as the result of some erroneous calculation (in which case we want to include it in the builder payload so the user gets an error instead of ignoring it).